test/routes.js

```diff
@@ -142,11 +142,7 @@ tape('POST /channel: should not work with invalid withdrawPeriodStart', async fu
 			...dummyVals.channel.spec,
 			withdrawPeriodStart: new Date('2200-01-01').getTime(),
 			minPerImpression: '1',
-			maxPerImpression: '1',
-			validators: [
```

Comment:
This seems to be removed because it was already in the `dummyVals.channel.spec`, correct?

Reply:
yes